Documentation for the webserver installation on Debian

winem opened the pull request:
The documentation was missing and it just said: 
> Debian / Ubuntu
> Your web server should be up and running after the installation.

Fixed this and added a short how-to test it.

dnsmichi commented:
That's for the reason that the Debian packages pull in Apache as dependency already. Didn't you have that on your system?

winem commented:
Yes, you are right. It's a dependency and already installed when you hit the point in the documentation. So I just misread the documentation. 
I reverted my change and added a small addition to make it a bit more clear I think. But it's no longer that important I think. You just have to be a bit careful when you read the documentation. 

Thanks for the feedback!

lippserd commented:
Hi,

Thanks for the small change. I'll merge it.

Cheers,
Eric
